man/InitDeviceTcti.3.in

```diff
@@ -1,26 +1,21 @@
 .\" Process this file with
 .\" groff -man -Tascii foo.1
 .\"
-.TH InitDeviceTcti 3 "JUNE 2017" Intel "TPM2 Software Stack"
+.TH Tss2_Tcti_Device_Init 3 "MARCH 2018" Intel "TPM2 Software Stack"
 .SH NAME
-InitDeviceTcti \- Initialization function for the device TCTI library.
+Tss2_Tcti_Device_init \- Initialization function for the device TCTI library.
 .SH SYNOPSIS
 .B #include <tcti/tcti_device.h>
 .sp
-.nf
-typedef struct {
-    const char *device_path;
-} TCTI_DEVICE_CONF;
-.fi
 .sp
-.BI "TSS2_RC InitDeviceTcti (TSS2_TCTI_CONTEXT " "*tctiContext" ", size_t " "*contextSize" ", const TCTI_DEVICE_CONF " "*config" ");"
+.BI "TSS2_RC Tss2_Tcti_Device_Init (TSS2_TCTI_CONTEXT " "*tctiContext" ", size_t " "*contextSize" ", const char " "*config" ");"
 .sp
```

Comment:
I think the second parameter is now just called `size` in the header.

Reply:
I'm on it